frontend/src/components/MonacoEditor.tsx

```diff
@@ -99,6 +100,7 @@ const MonacoEditor = ({ content, filename }: Props) => {
       </div>
       <div style={{ fontSize: 14, marginTop: 5, marginBottom: 5 }}>
         {(!user || !user.me) && 'Please login to enable saving'}
+        {(user && user.me && currentBranch && currentBranch !== '') && `On branch ${currentBranch}`}
```

Comment:
`(user && user.me && currentBranch && currentBranch !== '')` Can this be done in simpler way? Pretty hard to understand what is it checking.

Reply:
```suggestion
        {user?.me && currentBranch && `On branch ${currentBranch}`}
```
I don't really have time to test it but I think this should do the same thing. Empty string is "falsy" so checking `currentBranch` already returns false if the branch name is empty.